packages/react-components/src/components/LocalVideoTile.tsx

```diff
@@ -192,10 +200,28 @@ export const _LocalVideoTile = React.memo(
       showCameraSwitcherInLocalPreview
     ]);
 
-    const reactionOverlay =
-      reactionResources !== undefined ? (
-        <MeetingReactionOverlay overlayMode="grid-tiles" reaction={reaction} reactionResources={reactionResources} />
-      ) : undefined;
+    const videoTileOverlay = useMemo(() => {
+      const reactionOverlay =
+        reactionResources !== undefined ? (
+          <MeetingReactionOverlay overlayMode="grid-tiles" reaction={reaction} reactionResources={reactionResources} />
+        ) : undefined;
+      // If there are only one particiapnt in the call, show the waiting for others to join message
+      if (props.participantsCount === 1) {
+        return (
+          <>
+            {reactionOverlay}
+            <Stack className={mergeStyles(videoContainerStyles, overlayStyles())}>
```

Comment:
Can we not have overlayStyles like videoContainerStyles?
```
export const videoContainerStyles: IStyle = {
  position: 'absolute',
  top: '0',
  left: '0',
  width: '100%',
  height: '100%',
  minWidth: '100%',
  minHeight: '100%',
  objectPosition: 'center',
  objectFit: 'cover',
  zIndex: 0
};
```

instead of
```

/**
 * @private
 */
export const overlayStyles = (): IStyle => {
  return {
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    backgroundColor: 'rgba(0, 0, 0, 0.6)'
  };
};
```

Reply:
We need both here, by using this method we can avoid duplicate code